src/payloads/add_sticker_to_set.rs

```diff
@@ -0,0 +1,30 @@
+// This file is auto generated by `cg` <https://github.com/teloxide/cg> (455146e).
+// **DO NOT EDIT THIS FILE**,
+// edit `cg` instead.
+use serde::Serialize;
+
+use crate::types::{InputSticker, MaskPosition, True};
+
+impl_payload! {
+    /// Use this method to add a new sticker to a set created by the bot. Animated stickers can be added to animated sticker sets and only to them. Animated sticker sets can have up to 50 stickers. Static sticker sets can have up to 120 stickers. Returns _True_ on success.
+    #[derive(Debug, PartialEq, Clone, Serialize)]
+    pub AddStickerToSet (AddStickerToSetSetters) => True {
+        required {
+            /// User identifier of sticker file owner
+            pub user_id: u32,
```

Comment:
`user_id` is `i32` according to [Telegram Bot API](https://core.telegram.org/bots/api#addstickertoset), isn't it?

Reply:
Ah, good catch